doc/source/_toc.yml

```
@@ -7,7 +7,37 @@ parts:
       - file: ray-overview/installation
       - file: ray-overview/ray-libraries
 
-  - caption: Ray ML
+  - caption: Ray AI Runtime
+    chapters:
+    - file: ray-air/getting-started
+      title: "What is Ray AI Runtime (AIR)?"
+    - file: ray-air/key-concepts
+    - file: ray-air/user-guides
+      sections:
+        - file: ray-air/check-ingest
+        - file: ray-air/preprocessors
+        - file: ray-air/examples/analyze_tuning_results
+          title: "Analyze hyperparameter tuning results"
+        - file: ray-air/deployment
+        - file: ray-air/examples/upload_to_comet_ml
+        - file: ray-air/examples/upload_to_wandb
+    - file: ray-air/examples/index
+      sections:
+        - file: ray-air/examples/torch_image_example
+        - file: ray-air/examples/tfx_tabular_train_to_serve
+          title: "Tabular Data Training and Serving with Keras"
+        - file: ray-air/examples/huggingface_text_classification
+          title: "Fine-tune a 🤗 Transformers model"
+        - file: ray-air/examples/sklearn_example
+        - file: ray-air/examples/xgboost_example
+        - file: ray-air/examples/lightgbm_example
+        - file: ray-air/examples/torch_incremental_learning
+        - file: ray-air/examples/rl_serving_example
+        - file: ray-air/examples/rl_online_example
+        - file: ray-air/examples/rl_offline_example
+    - file: ray-air/package-ref
+
+  - caption: Ray Libraries
```

```suggestion
  - caption: AIR Libraries
```

This has to say AIR libraries, both to emphasize the connection with AIR, and to avoid needing to introduce Ray Libraries as a new top-level concept.

---

Good point.